components/formats-gpl/src/loci/formats/in/LIFReader.java

```diff
@@ -126,7 +126,7 @@ private static ImmutableMap<String, Integer> createChannelPriorities() {
   // -- Fields --
 
   /** Offsets to memory blocks, paired with their corresponding description. */
```

Comment:
Could be updated  to something along the lines of `Memory blocks including offsets and ID`

Reply:
Done in 18ed364dcc.